.devcontainer/devcontainer.json

```diff
@@ -67,7 +67,7 @@
 	// Use 'forwardPorts' to make a list of ports inside the container available locally.
 	// "forwardPorts": [],
 	// package is uninstalled from the container image after it is created to avoid circular import errors during development 
-	"postCreateCommand": "sudo pip3 uninstall swxsoc -y",
+	"postCreateCommand": "sudo pip3 uninstall swxsoc hermes_core -y",
```

Comment:
why do we need to uninstall hermes_core?

Reply:
At the moment it is using the hermes development base image, which has hermes_core installed. Wanted to ensure no dependency issues.